Justfile

```diff
@@ -96,9 +96,9 @@ all-release: build-release test-release
 #---------------#
 
 # If you're not cafkafk and she isn't dead, don't run this!
-# 
```

Comment:
The formatting fixes should ideally be part of a different commit.

Reply:
Yeah my bad, sorry for that.
My editor automatically remove extra whitespaces before saving :sweat_smile:

Reply:
git allows for splitting things into multiple commits retroactively ... like a soft reset, then commiting the changes individually and then force pushing here to overwrite the single commit.

Reply:
I mean we could also do that for you, but that will surely take longer.

Reply:
Hope it's better after spliting and rewriting commit messages :)